dev/tidb-in-kubernetes/maintain/restart.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 重启 TiDB 集群
+category: how-to
+---
+
+# 重启 TiDB 集群
+
+本文描述了如何重启 Kubernetes 集群上的 TiDB 集群，包括重启某个 Pod，重启某个组件的所有 Pod 和 重启 TiDB 集群的所有 Pod。
```

Comment:
Which version the operator is for this document? For version 1.0.X, the pod could be restarted by `kubectl delete` which is easy but not safe. For version 1.1.X, when the admission pod webhook is enabled, users could restart pod by annotating pod which is safer.

Reply:
This is for v1.0.x, no safe way to delete pod for 1.0.x.
For 1.1.x, we need to update this document before it's released.

Reply:
ok. I think we should tell the user the risk in this document.

Reply:
I have added the graceful restart procedure.